### PR TITLE
Feature fr10 banner for supervisors

### PR DESCRIPTION
**Pull request by Liljenmaa**
A banner will show whenever a logged-in user has status "not confirmed" on some of their supervisions.

The banner will disappear if:
- The user logs off

The banner will reappear when:
- The user clicks on the TASERA logo
- The user refreshes the page (by the means of F5, changing the language)

The banner will NOT reappear when:
- Other actions mentioned before
- The user performs an action that does not refresh the page (browse weekview, click on the side bar options)

NOTE: Additional testing is useful. If you encounter anything that works oddly or something should work differently, please let me know.

### CODE REVIEW

**Comment by TuoLuo994** [comment]
I think the banner should stay visible at all times (not disappear when clicked, only when accepted/refused supervision), because confirming the supervision is a high priority task for the user, and even if the user doesn't want to do it right away, the alert-component is not too distracting.

**Comment by TuoLuo994** [comment]
With the original background coloring, the current info-severity warning is not visible enough. I would add `variant="filled"` and  change the severity to warning or even error.

File: front/src/navbar/Nav.js [hunk 1]
```diff
@@ -50,7 +50,75 @@ const elementStyle = {
   marginTop:10
 }
 
-const SideMenu = ({setName, superuser}) => {
+class SupervisorNotification extends Component {
```
[the remark this answers is not in the document]
Comment (Liljenmaa) [reply front/src/navbar/Nav.js]: Done

**Review by TuoLuo994 — CHANGES_REQUESTED** [review]
If I click on the banner and click cancel, the banner becomes unclickable.

**Comment by Liljenmaa** [comment]
Fixed the old reservations bug. PR requested.

**Review by TuoLuo994 — APPROVED** [review]

**Comment by TuoLuo994** [comment]
Looks good to me

**Pull request merged**